core/src/main/scala/kafka/server/BrokerServer.scala

```diff
@@ -221,7 +221,7 @@ class BrokerServer(
         config,
         channelName = "forwarding",
         threadNamePrefix,
-        retryTimeoutMs = 60000
+        retryTimeoutMs = config.brokerSessionTimeoutMs / 2
```

Comment:
why are we changing this one, given that this BrokerToControllerChannelManager does not send heartbeats?